test/helpers/messaging_service_helpers.go

```diff
@@ -190,6 +190,16 @@ func ForceDisconnectViaSEMPv2(messagingService solace.MessagingService) {
 	ExpectWithOffset(1, err).ToNot(HaveOccurred())
 }
 
+func CheckMessagingServiceDisconnectedViaSEMPv2(messagingService solace.MessagingService) {
```

Comment:
Should this method be used in the test that was updated?

Reply:
No, I wanted to use it but changed my mind because it was better to use a Service Interruption Listener to check for the session down event.

Reply:
I like the service interruption listener method, if this method is un-used we should remove it then

Reply:
I have removed the unused function @sdewilde .